css/css-flexbox/WEB_FEATURES.yml

```diff
@@ -0,0 +1,54 @@
+features:
+- name: flexbox
+  files: "**"
```

Comment:
Under the current logic:
In this current configuration: `flexbox` and `flexbox-gap` will have some overlapping features. Both will have `column-reverse-gap.html` (due to `flexbox`'s `**` and `flexbox-gap`'s explicit listing of `column-reverse-gap.html`)

Or are you expecting different behavior?

Reply:
I'm expecting that `flexbox-gap` will "steal" the features from `flexbox`, similar to how the recursive behavior works.